.travis.yml

```diff
@@ -10,9 +10,9 @@ branches:
 
 env:
   matrix:
-    - LUA=5.2 DO_CHECKQA=1 LUANAME=lua5.2 BUILD_APIDOC=true DO_COVERAGE=coveralls
+    - LUA=5.2 DO_CHECKQA=1 LUANAME=lua5.2 DO_COVERAGE=coveralls
     # luajit: installed from source.
-    - LUA=5.1 LUANAME=luajit-2.0 LUALIBRARY=/usr/lib/libluajit-5.1.so LUAROCKS_ARGS=--lua-suffix=jit-2.0.4 TEST_PREV_COMMITS=1
+    - LUA=5.1 LUANAME=luajit-2.0 LUALIBRARY=/usr/lib/libluajit-5.1.so BUILD_APIDOC=true LUAROCKS_ARGS=--lua-suffix=jit-2.0.4 TEST_PREV_COMMITS=1
```

Comment:
Seems to be not working as expected currently anyway: looks like it tests only one of them.

> its slow down is a problem for larger pull requests.

Well, I think that is OK, given the benefit of ensuring that git-bisect works (better).
And given that it uses the existing build (cache), it is probably not so slow after all.

My point in the comment above was just that this should be taken into account when shuffling the build matrix.